src/poker/game/GamePlay.java

```diff
@@ -51,11 +51,11 @@ public void gameStart() {
                     nonActivePlayer = temp;
                 }
                 lastAction = null;
-
                 activePlayer = bigBlindPlayer;
                 nonActivePlayer = smallBlindPlayer;
                 if (handleActionResult.isFold || handleActionResult.isAllIn) {
                     if (handleActionResult.isAllIn){
+                        actionHandler.handlePrintEquity(i);
                         gameState.setGamePhase(GamePhase.RIVER);
                         actionHandler.handleNextStreet(gameState,activePlayer,nonActivePlayer);
```

Comment:
ogolnie to nie podoba mi sie ze handlujemy akcje dla allina tutaj to wszystko powinno byc juz zrobione w handleAction()